doc/source/user/basics.indexing.rst

```diff
@@ -176,8 +178,9 @@ concepts to remember include:
       >>> x[5:]
       array([5, 6, 7, 8, 9])
 
-- If the number of objects in the selection tuple is less than
-  *N*, then ``:`` is assumed for any subsequent dimensions.
+- If the length of the selection object in the square brackets
+  is less than the *N* dimensions of the array, then ``:`` is assumed
+  for any remaining dimensions.
```

Comment:
I prefer the original wording here. I think using "selection object" actually makes this slightly less clear in that the section as a whole is dealing with basic indexing, which includes scalar indices, e.g. (`x[5]`). Talking about the "length of the selection tuple" makes it clear that this statement applies specifically to indexing multiple dimensions. The proposed wording IMO introduces an ambiguity in the case of scalar indices (which of course are objects).

Reply:
How about just saying "the length of the selection", following the use of this expression in the sections before this one? I think the main concern here is mentioning a tuple when in fact the example shows a slice.

Reply:
Maybe the example itself should be updated?

```rst
":" is assumed for the non-specified dimensions of the shape, so ``x[1:]``
is equivalent to (but more convenient to write) than ``x[1:, :, :]``::

    >>> x[1:]
    array([[[4],
            [5],
            [6]]])
```